core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review.directive.html

```diff
@@ -62,4 +65,9 @@ <h3>Review translation</h3>
   .oppia-suggestion-resolve-button {
     min-width: 165px;
   }
+  @media only screen and (max-width: 1023px) {
```

Comment:
How was this max width chosen?

Reply:
1023px appeared to be a breakpoint for the modal, actually - that's why I chose it. After this breakpoint, the heading branches into multiple lines from a single line. Should I add a comment that makes this easier to understand?

Reply:
I have added a comment explaining this. It should be clear now.